Sources/Shared/Models/TeamMember.swift

```diff
@@ -26,7 +26,7 @@
 
 import Foundation
 
-public class TeamMember: VIMModelObject, ConnectionsProviding, ConnectionsParsing {
+@objcMembers public class TeamMember: VIMModelObject, ConnectionsProviding, ConnectionsParsing {
```

Comment:
I think it's usually placed in a new line? Not 100% sure. But that's how I've seen it in the past with other classes.